### ISSUE TITLE
case sensitivity issue in cmakelists.txt

### ISSUE THREAD

**Issue by meshula** [OpenImageIO/oiio#1682]
This line in Cmakelists.txt -
````
file (RENAME "${CMAKE_BINARY_DIR}/cpack/README.md" "${CMAKE_BINARY_DIR}/cpack/Readme.md")
````
causes problems on recent macOS versions because the RENAME command complains that it is invalid to rename a file such that only case differs.

I thought maybe the rename could be done on the COPY in the previous line, but no.

**Comment by lgritz** [comment]
After all this time, it's not clear to me why we change case here. I'm totally fine with it staying README.md. Can you think of a reason why we should try to do anything more complicated? Was it required to be "Readme.md" for some reason?

**Comment by meshula** [comment]
It looks historical. Later in the file, a CPACK_RESOURCE file name is set to the non-shouting version of Readme, and the text Readme is not referenced elsewhere in the source tree. So I think if the later reference is changed to should README.md it'll all be cool.

**Comment by lgritz** [comment]
Fix proposed in #1687
